### PR TITLE
Circuits for factorized UCCSD (fUCCSD)

### PR DESCRIPTION
**Pull request by erikkjellgren**

### CODE REVIEW

File: tests/test_qiskit_interface.py [hunk 1]
```diff
@@ -159,7 +160,7 @@ def test_LiH_dumb_projected_estimator() -> None:
     estimator = Estimator()
     mapper = ParityMapper(num_particles=(1, 1))
 
-    QI = QuantumInterface(estimator, "ErikSD", mapper)
+    QI = QuantumInterface(estimator, "tUCCSD", mapper)
```
Comment (ziechys) [inline tests/test_qiskit_interface.py:163]: I have used ErikSD here to speed up the tests. But up to you.

File: tests/test_qiskit_interface.py [hunk 2]
```diff
@@ -802,3 +803,36 @@ def test_qiskit_aer() -> None:
...
+    SQobj = sq.SlowQuant()
+    SQobj.set_molecule(
+        """O   0.0  0.0           0.1035174918;
+    H   0.0  0.7955612117 -0.4640237459;
+    H   0.0 -0.7955612117 -0.46402374590;""",
+        distance_unit="angstrom",
+    )
+    SQobj.set_basis_set("STO-3G")
+    SQobj.init_hartree_fock()
+    SQobj.hartree_fock.run_restricted_hartree_fock()
```
Comment (ziechys) [inline tests/test_qiskit_interface.py:819]: Is there a reason you don't use PySCF anymore as in the other tests?

File: slowquant/qiskit_interface/operators_circuits.py [hunk 3]
```diff
@@ -2,6 +2,33 @@
 from qiskit.circuit import Parameter, QuantumCircuit
 
 
+def f2q(i: int, num_orbs: int) -> int:
```
Comment (ziechys) [inline slowquant/qiskit_interface/operators_circuits.py:5]: Maybe pedantic, but: Isn't this more a util.py function?

**Review by ziechys — APPROVED** [review]

File: tests/test_qiskit_interface.py (hunk 1 above)
Comment (ziechys) [reply tests/test_qiskit_interface.py:163]: I just saw that you removed this Ansatz altogether. RIP :D

Comment (erikkjellgren) [reply tests/test_qiskit_interface.py:163]: Ye, I rather just get rid of "dead" code.
I think we have moved on from it xd

**Pull request merged**